src/tests/ftest/container/multiple_delete.py

```diff
@@ -61,7 +61,11 @@ def test_multiple_container_delete(self):
         self.log.info("SCM = %d, SSD = %d", final_scm_fs, final_ssd_fs)
 
         self.log.info("Verifying SSD space is recovered")
-        self.check_pool_free_space(self.pool, expected_nvme=initial_ssd_fs)
+        try:
+            self.pool.check_free_space(expected_nvme=initial_ssd_fs)
+        except DaosTestError as error:
```

Comment:
(pylint-undefined-variable) Undefined variable 'DaosTestError'

Reply:
Probably missing import.  I see this in other files
from general_utils import DaosTestError